scania-digital-design-system/tegel
# revert(table): passing on unit value with horizontalscroll (#736)

This reverts commit 7d584a1c8a72ffe8814ba67ea36aab81cbaa53a4.

## **Describe pull-request**  
Reverting erroneus PR introduced [here](https://github.com/scania-digital-design-system/tegel/pull/736)

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
Run ```npm run build-all```, and see that it compiles all packages.
